stats/compute_test.go

```diff
@@ -97,11 +106,16 @@ func TestCompute(t *testing.T) {
 //
 // The returned dataset offers predictable output when computing common
 // statistics such as min, max, mean, etc.
-func newDataStub(size int) stats.Float64Data {
+func newDataStub(size int) []float64 {
 	floats := make([]float64, size)
 
 	for i, v := range rand.Perm(size) {
 		floats[i] = float64(v + 1)
 	}
 	return floats
 }
+
+// inRange returns true if val is equal to target with a margin of error.
+func inRange(val, target, margin float64) bool {
```

Comment:
[nitpick] `inRange` is a bit misleading as we would expect a signature like `func(v, min, max float64) bool`, which is the same of the current one in terms of types.
```suggestion
func getBooleanValueRepresentingWhetherValIsSuperiorOrEqualToTargetMinusMarginOrInferiorOrEqualToTargetPlusMarginOrWhetherItIsNot(val, target, margin float64) bool {
```

Or something like `approxEqual`, `equalWithMargin`? Not 100% satisfying but probably less error prone. (Anyway it's really not critical here)